blog/views.py

```diff
@@ -0,0 +1,14 @@
+from django.views import generic
+
+from blog.models import User, Post, Commentary
+
+
+class PostListView(generic.ListView):
+    model = Post
+    template_name = "blog/index.html"
+    paginate_by = 5
+
+
+class PostDetailView(generic.DetailView):
+    model = Post
+    template_name = "blog/post_detail.html"
```

Comment:
Ensure that `PostDetailView` includes functionality to display comments related to the post and a form for adding new comments. This is essential for meeting the requirements of the task.